tests/lax_numpy_test.py

```diff
@@ -1996,6 +1996,25 @@ def testRoll(self, shape, dtype, shifts, axis, rng_factory):
     self._CheckAgainstNumpy(jnp_op, onp_op, args_maker, check_dtypes=True)
     self._CompileAndCheck(jnp_op, args_maker, check_dtypes=True)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
```

Comment:
I am wondering if this will fail with numerical tolerances on TPUs. I'll import later today.